src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -90,17 +105,42 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
             return false;
         }
 
+        /// <summary>Find the cgroup memory limit path.</summary>
+        /// <returns>The limit path if found; otherwise, null.</returns>
+        private static string FindCGroupMemoryLimitPath()
+        {
+            string cgroupMemoryPath = FindCGroupPath("memory", out CGroupVersion version);
+            if (cgroupMemoryPath == null)
+            {
+                return null;
+            }
+
+            if (version == CGroupVersion.CGroup1)
+            {
+                return cgroupMemoryPath + "/memory.limit_in_bytes";
+            }
+            else if (version == CGroupVersion.CGroup2)
```

Comment:
Nit: this `else` isn't necessary, and it'd better follow the rest of the method's pattern if this were instead just an `if` with a blank line above it